mmv1/products/notebooks/api.yaml

```diff
@@ -245,6 +254,31 @@ objects:
             required: true
             description: |
               Count of cores of this accelerator.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'shieldedInstanceConfig'
```

Comment:
Same as above- will this block be returned by the API even if the user has not specified it?

Reply:
Yes, if user does not specify it, API returns enableIntegrityMonitoring and enableVtpm set as True.
By changing default_value to true would this fix it?

Reply:
Not quite. Those will fix it while the block is defined, but not if it isn't. You can apply `default_from_api: true` to the field in `terraform.yaml` to handle this case. https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/notebooks/terraform.yaml#L76-L77 is an example on another field within this resource.

Reply:
Since the fields are nested would this do it:

```
      enableIntegrityMonitoring: !ruby/object:Overrides::Terraform::PropertyOverride
        default_from_api: true
      enableVtpm: !ruby/object:Overrides::Terraform::PropertyOverride
        default_from_api: true
```

Reply:
Sorry, I should have been more clear! You only need to apply it to the top-level field `shieldedInstanceConfig`